security.md

```diff
@@ -4,22 +4,33 @@ The Simeon software runs securely in your own Azure DevOps environment, tied dir
 
 Because Simeon runs in your own Azure DevOps environment, you are in complete control of the security. You can harden access to your Azure DevOps environment as much as you like.
 
-Our Web Admin user interface connects directly to your Azure DevOps environment and none of your data ever passes through our servers.
+Our Web Admin user interface is a single page application that runs entirely in your browser and connects directly to your Azure DevOps environment. None of your data ever passes through our servers. The Web Admin is an Azure Static Web App with zone redundant storage. All traffic is secured via HTTPS with TLS 1.2.
```

Comment:
We say "none of your data ever passes through our servers"

Do we want to speak to our serverless technology, etc?

Reply:
The single page application part conveys that